File: Python/no. checking/spy.py
#Spy no.
n=int(input("Enter the no. = "))
s=0
p=1
x=n
while(n!=0):
    s+=n%10
    p*=n%10
    n//=10
if s==p:
    print("Spy No.")
else:
    print("not Spy No.")

File: C Programms/No. Checking/spy.c
//WAp to check Spy No.
#include<stdio.h>
void main()
{
    int n,x,s=0,p=1;
    printf("Enter the no. = ");
    scanf("%d",&n);
    for(x=n;n!=0;n/=10)
    {
        s+=n%10;
        p*=n%10;
    }
    (s==p)?printf("%d is a Spy no.",n):printf("%d is ot a Spy no.",n);
}